BootloaderCommonPkg/BootloaderCommonPkg.dec

```diff
@@ -179,6 +179,7 @@
   gPlatformCommonLibTokenSpaceGuid.PcdBuildSmmHobs                     |        0x03| UINT8 | 0x20000705
   gPlatformCommonLibTokenSpaceGuid.PcdTccEnabled                  | FALSE  | BOOLEAN | 0x20000221
   gPlatformCommonLibTokenSpaceGuid.PcdEnableCryptoPerfTest        | FALSE  | BOOLEAN | 0x20000224
+  gPlatformCommonLibTokenSpaceGuid.PcdElfSupportDisabled          | FALSE  | BOOLEAN | 0x2000022A
```

Comment:
Prefer using Feature PCD type. Rename to PcdElfSupportEnabled, and set to true by default.

Reply:
Updated